opendc-simulator/opendc-simulator-compute/src/main/java/org/opendc/simulator/compute/SimPsuFactories.java

```diff
@@ -57,8 +58,8 @@ public static SimPsuFactory noop() {
      *
      * @param model The power model to estimate the power consumption based on the CPU usage.
      */
-    public static SimPsuFactory simple(CpuPowerModel model) {
-        return (machine, graph) -> new SimplePsu(graph, model);
+    public static SimPsuFactory simple(CpuPowerModel model, ThermalModel thermalModel) {
```

Comment:
Ideally, we would make the thermalModel variable nullable. 
Currently, you always have to put in a thermalModel. 
This is not ideal.